packages/widgets/src/lib/components/InferenceWidget/shared/WidgetSubmitBtn/WidgetSubmitBtn.svelte

```diff
@@ -6,21 +6,8 @@
 	export let isLoading: boolean;
 	export let label = "Compute";
 	export let onClick: () => void;
-
-	function onKeyDown(e: KeyboardEvent) {
-		if (isLoading || isDisabled) {
-			return;
-		}
-		// run inference on cmd+Enter
-		if (e.code === "Enter" && (e.metaKey || e.ctrlKey)) {
-			e.preventDefault();
-			onClick();
-		}
-	}
 </script>
 
-<svelte:window on:keydown={onKeyDown} />
```

Comment:
This listener has not been replaced by `use:runShortcut` on the `button` element, is that intended?

Reply:
yep. that's the entire point of this PR to ["localize"](https://github.com/huggingface/huggingface.js/pull/577#issue-2205949366)